export/securedrop_export/print/service.py

```diff
@@ -241,29 +241,29 @@ def _is_open_office_file(self, filename):
     def _print_file(self, file_to_print):
         # If the file to print is an (open)office document, we need to call unoconf to
         # convert the file to pdf as printer drivers do not support this format
-        if self._is_open_office_file(file_to_print):
-            logger.info("Converting Office document to pdf")
-            folder = os.path.dirname(file_to_print)
-            converted_filename = file_to_print + ".pdf"
-            converted_path = os.path.join(folder, converted_filename)
-            self.safe_check_call(
-                command=["unoconv", "-o", converted_path, file_to_print],
-                error_status=Status.ERROR_PRINT,
+        try:
```

Comment:
I personally prefer having narrow try/except blocks so that they just cover the exact lines that need to be covered, which in this case would be the two `subprocess.check_call` functions. It adds a little bit of duplication in the exception handler, but I think it makes the intent of the code cleaner, and prevents things from being accidentally over-covered during refactoring.

(This is also what the pre-existing code did via the safe_check_call function.)

Reply:
Yup that's fine with me. I went back and forth because we always report ERROR_PRINT, but I can make the change